packages/cactus-test-cmd-api-server/src/test/typescript/integration/plugin-import-with-npm-install.test.ts

```diff
@@ -56,6 +56,7 @@ test("can instal plugins at runtime based on imports", async (t: Test) => {
       options: {
         instanceId: uuidv4(),
         keychainId: uuidv4(),
+        version: "0.9.0",
```

Comment:
@elenaizaguirre Can you add a test assertion that verifies if this particular version has been installed? You can parse the package.json file on the file-system where the plugin was installed and then check for strict equality between the version inside the package.json and what you declared here

Reply:
@petermetz Done, I added `--save-exact` flag to api-server.ts installation command. I have created another independent test for this and have reverted the others.

Reply:
@elenaizaguirre Thank you for adjusting! 

One small nit-pick: It can be done without the `--save-exact` option if you parse the package.json of the installed plugin package instead of the project level package.json:
Meaning that when you run the install, it creates two package.json files, one as `./package.json` (relative to the plugins folder) and this is the one you are parsing now, but there's a second one under `./node_modules/@hyperledger/cactus-some-plugin-package/package.json`

If you parse the second file and check the `"version"` property it should have the expected `0.9.0` there and this way you don't need the save-exact option which is a minor thing but I'm concerned about using it due to unforeseen side-effects that it may cause (remember how much trouble we had with the npm installation mechanism already)